app/views/recipes/edit.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class='container wrapper'>
   <div class='row'>
     <div class='col-md-offset-3 col-md-6 col-md-offset-3'>
-      <%= form_for(@recipe, url: recipe_path) do |i| %>
+      <%= form_for @recipe do |i| %>
```

Comment:
Please be more literate when naming stuff, `i` does not tell anything. Something like `form` would be good.